pkg/gadgets/top/tcp/types/types.go

```diff
@@ -49,7 +49,7 @@ type Stats struct {
 
 	Pid       int32  `json:"pid,omitempty" column:"pid,template:pid"`
 	Comm      string `json:"comm,omitempty" column:"comm,template:comm"`
-	IPVersion uint16 `json:"ipversion,omitempty" column:"ip,template:ipversion"`
+	IPVersion int    `json:"ipversion,omitempty" column:"ip,template:ipversion"`
```

Comment:
Although we don't transmit this in binary form, yet, IMHO we should always use types with a fixed size (that aren't platform dependent) for things that can be exchanged between two different systems.